app/decorators/application_plan_decorator.rb

```diff
@@ -0,0 +1,45 @@
+# frozen_string_literal: true
+
+class ApplicationPlanDecorator < ApplicationDecorator
+
+  def link_to_edit(**options)
+    h.link_to(name, h.edit_admin_application_plan_path(self), options)
+  end
+
+  def link_to_applications(**options)
+    live_applications = cinstances.live
+    text = h.pluralize(live_applications.size, 'application')
+
+    if h.can?(:show, Cinstance)
+      h.link_to(text, plan_path, options)
+    else
+      text
+    end
+  end
+
+  def plan_path
+    service = context.fetch(:service) { object.service }
+    h.admin_service_applications_path(service, search: { plan_id: id })
+  end
+
+  def index_table_data
```

Comment:
Will this be present in the responses of the Admin API as well?

Reply:
No, this is exclusive for the view